content/en/docs/zero-code/java/quarkus/getting-started.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Getting started with Quarkus
+weight: 20
+---
+
+To enable OpenTelemetry in your Quarkus application, add the
+`quarkus-opentelemetry` extension dependency to your project.
+
+{{< tabpane >}} {{< tab header="Maven (`pom.xml`)" lang=Maven >}}
+
+````xml
+
+{{< tabpane text=true >}} {{% tab header="Maven (`pom.xml`)" lang=Maven %}}
+
```

Comment:
This second and nested `tabpane ` seems to have been added by mistake (and it caused an extra escaping of the xml):

```suggestion
```